bot.yml

```diff
@@ -26,7 +26,7 @@ config:
    type: secret
    default: xxx
    global: true
- - name: FILENAME
+ - name: FILE_NAME
    friendly_name: name of cached data file
```

Comment:
naming convention for environment variables should be with underscore.. please check: https://google.github.io/styleguide/shell.xml#Naming_Conventions